src/GSoft.Authentication.ClientCredentialsGrant.Tests/IntegrationTests.cs

```diff
@@ -137,4 +139,58 @@ public async Task Real_Client_Server_Communication()
             cts.Cancel();
         }
     }
+
+    private sealed class TestServerClient : DelegatingHandler
```

Comment:
Maybe rename this more like `TestServerHandler` instead?

Reply:
ahh yeah good point!